Godeps/Godeps.json

```diff
@@ -10,6 +10,10 @@
 			"Rev": "70b2c90b260171e829f1ebd7c17f600c11858dbe"
 		},
 		{
+			"ImportPath": "github.com/microsoft/hcsshim",
+			"Rev": "fc8f843b468326d898f2ef2257440c792d2fa283"
+		},
+		{
```

Comment:
Can you import at the latest HCSShim - currently 35ad4d808a97203cb1748d7c43167e91f51e7f86 (already in docker master)